its/plugin/pom.xml

```diff
@@ -3,14 +3,13 @@
   xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
   <parent>
-    <groupId>org.sonarsource.parent</groupId>
-    <artifactId>parent</artifactId>
-    <version>23</version>
+    <groupId>org.sonarsource.android</groupId>
+    <artifactId>sonar-android</artifactId>
+    <version>1.2-SNAPSHOT</version>
+    <relativePath>../..</relativePath>
```

Comment:
By default it looks the parent to `../pom.xm` on the file system (which is not the case there), otherwise it looks in the local repo (but we don't use the `install` goal), and lastly in the remote repo (but it has not be deployed yet) : http://maven.apache.org/ref/3.0/maven-model/maven.html#parent